src/index.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react'
 import * as ReactDOM from 'react-dom'
+import 'bootstrap/dist/css/bootstrap.css';
```

Comment:
I wonder if this imports the whole Bootstrap library or if Webpack is smart enough to only output the classes that are actually used. @michaellouviere, do you know?